index.src.html

```diff
@@ -332,6 +332,32 @@ <h3 dfn export id="referrer-policy-origin" oldids="referrer-policy-state-origin"
     priori</em> authenticated URLs</a>.
   </div>
 
+  <h3 dfn export id="referrer-policy-safe-origin" oldids="referrer-policy-state-safe-origin">"<code>safe-origin</code>"</h3>
+
+  The <a>"<code>safe-origin</code>"</a> policy sends the
+  <a lt="ASCII serialization of an origin">ASCII serialization</a> of the
+  <a>origin</a> of the <a>request client</a> when making requests from
+  <a>TLS-protected</a> <a>environment settings object</a> to a
+  <a><em>a priori</em> authenticated URL</a>, and requests from
+  <a>request clients</a> which are <em>not</em> <a>TLS-protected</a> to any
+  <a>origin</a>.
+
+  Requests from <a>TLS-protected</a> <a>request clients</a> to non-<a><em>a
+  priori</em> authenticated URLs</a>, on the other hand, will contain no
+  referrer information. A <code><a>Referer</a></code> HTTP header will not be
+  sent.
+
+  <div class="example">
+    If a document at <code>https://example.com/page.html</code> sets a policy of
+    <a>"<code>safe-origin</code>"</a>, then navigations to
+    <code>https://<strong>not</strong>.eaxmple.com</code> would send a
+    <a><code>Referer</code></a> header with a value of
+    <code>https://example.com/</code>. This explicitly excludes navigations to
+    URLs that are not <a><em>a priori</em> authenticated URLs</a>.
+    Navigations to <code><strong>http://not</strong>.example.com</code> would
+    send no <a><code>Referer</code></a> header.
```

Comment:
Could you please include another example of what happens with requests from http://example.com?